youtube_dl/extractor/porn91.py

```diff
@@ -36,8 +36,8 @@ def _real_extract(self, url):
             raise ExtractorError('91 Porn says: Daily limit 10 videos exceeded', expected=True)
 
         title = self._search_regex(
-            r'<div id="viewvideo-title">([^<]+)</div>', webpage, 'title')
-        title = title.replace('\n', '')
+            r'<h4 class="login_register_header"[^>]+>([^<]+)</h4>', webpage, 'title')
```

Comment:
`<h4 class="login_register_header"[^>]+>` matches multiple times and does not properly identify title.